src/webgpu/api/validation/queue/submit.spec.ts

```diff
@@ -53,10 +70,52 @@ g.test('command_buffer,duplicate_buffers')
     `
   )
   .fn(t => {
-    const encoder0 = t.device.createCommandEncoder();
-    const cb = encoder0.finish();
+    const cb = t.createCommandBuffer();
 
     t.expectValidationError(() => {
       t.device.queue.submit([cb, cb]);
     }, true);
   });
+
+g.test('command_buffer,submit_invalidates')
```

Comment:
nit: split this into three tests (some of them are more basic than others, and likely to pass already in browsers)

Reply:
Done.